packages/core/src/modules/credentials/CredentialState.ts

```diff
@@ -8,6 +8,8 @@ export enum CredentialState {
   ProposalReceived = 'proposal-received',
   OfferSent = 'offer-sent',
   OfferReceived = 'offer-received',
+  OfferDeclinedReceived = 'offer-declined-received',
+  OfferDeclinedSent = 'offer-declined-sent',
```

Comment:
Given issue-credential v1, this would be an internal state as we won't send anything to the issuer when declining an offer (such as a problem report), so I think we want one state here--I'd recommend `OfferDeclined = 'offer-declined'`.

Reply:
I'm thinking whether we can go even more basic, and just store state `declined`? This way it can also be used for the issuer side, and other states. We can then also make `declineOffer` more generic, as it works for other states than `offer-received`.

Just a thought, I'm also fine with using `offer-declined` state.

Reply:
I'm in favor of that--so you're thinking no state validation for the `declineOffer` method since the credential can be in any state?